newsfragments/3133.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed socket module for some older systems which lack `socket.AI_NUMERICSERV`.

```

Comment:
Oops CI is still complaining about being unable to find `socket.AI_NUMERICSERV`. Try this instead:
```suggestion
Fixed socket module for some older systems which lack ``socket.AI_NUMERICSERV``.
```